app/controllers/articles_controller.rb

```diff
@@ -0,0 +1,199 @@
+# frozen_string_literal: true
```

Comment:
I should have renamed `article_controller` instead of creating a new file.
- Renaming allows a better diff (at least in GitHub). 
- Also one less chore (deleting the old file).